src/macros.rs

```diff
@@ -1,3 +1,40 @@
+/// Create an [`IndexMap`][crate::IndexMap] from a list of key-value pairs  
+/// and a `BuildHasherDefault`-wrapped custom hasher.  
```

Comment:
Trailing whitespace crept in here:
```suggestion
/// Create an [`IndexMap`][crate::IndexMap] from a list of key-value pairs
/// and a `BuildHasherDefault`-wrapped custom hasher.
```

I'm surprised `rustfmt` doesn't try to change that, but I guess it's something to do with markdown line-wrapping. We don't want to _force_ rendering in this particular way.